lib/loggers/index.js

```diff
@@ -60,21 +60,30 @@ container.add = function(id) {
   if(!existing) {
     const wrapper = Object.create(logger);
     wrapper.log = function(level, msg, ...meta) {
+      // get default meta to ensure it is always applied
+      const {defaultMeta} = this;
+      // use `Object.getPrototypeOf` because the prototype may change
+      // while the logging subsystem initializes
+      const log = Object.getPrototypeOf(wrapper).log;
       if(arguments.length === 1) {
-        return Object.getPrototypeOf(wrapper).log.apply(
-          wrapper, [level]);
+        return log.apply(wrapper, [{...defaultMeta, ...level}]);
```

Comment:
Thanks for diving into this.

thanks to the vscodes nodejs debugger, I was able to determine that `defaultMeta` is being injected into this code path already, which is why it was working in this case.  However, it must be manually added as you have done on L71.

https://github.com/winstonjs/winston/blob/master/lib/winston/create-logger.js#L80